### PR TITLE
Task INJIMOB-1568

### PR DESCRIPTION
**Pull request by santhoshsunder**
adding descriptions to the mimoto API scenarios

### CODE REVIEW

File: apitest/src/main/resources/mimoto/AidGetIndividualId/AidGetIndividualId.yml [hunk 1]
```diff
@@ -54,7 +56,8 @@ AidGetIndividualId:
 }'
 
    Mimoto_AidGetIndividualId_Getting_With_Invalid_Otp_Neg:
-      endPoint: /v1/mimoto/aid/get-individual-id
+      endPoint: v1/mimoto/aid/get-individual-id
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/mimoto/AidGetIndividualId/AidGetIndividualId.yml:59]: Endpoint shouldn't be changed here

File: apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml [hunk 2]
```diff
@@ -59,7 +61,8 @@ AuthLock:
     }
 }'
    Mimoto_AuthLock_with_invalidUIN_Valid_Otp_sid:
-      endPoint: /v1/mimoto/req/auth/lock
+      endPoint: //v1/mimoto/req/auth/lock
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml:64]: Endpoint shouldn't be changed here

File: apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml [hunk 3]
```diff
@@ -150,7 +155,8 @@ AuthLock:
     }
 }'
    Mimoto_AuthLock_invalidVID_Valid_Otp:
-      endPoint: /v1/mimoto/req/auth/lock
+      endPoint: mimoto/AuthLock/createAuthLock
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml:158]: Endpoint shouldn't be changed here

File: apitest/src/main/resources/mimoto/AuthUnLock/createAuthUnLock.yml [hunk 4]
```diff
@@ -1,6 +1,7 @@
 AuthUnlock:
    Mimoto_AuthUnLock_Valid_UIN_All:
-      endPoint: /v1/mimoto/req/auth/unlock
+      endPoint: v1/mimoto/req/auth/unlock
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/mimoto/AuthUnLock/createAuthUnLock.yml:3]: Endpoint shouldn't be changed here

File: apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml [hunk 5]
```diff
@@ -314,7 +330,8 @@ BindingOtp:
 }'
 
    Mimoto_BindingOtp_UIN_Future_RequestTime_Neg:
-      endPoint: /v1/mimoto/binding-otp
+      endPoint: /v1/mimoto/binding-ot
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml:333]: Endpoint shouldn't be changed here

File: apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml [hunk 6]
```diff
@@ -674,7 +708,8 @@ BindingOtp:
 }'
 
    Mimoto_BindingOtp_vid_Missing_RequestTime_Neg:
-      endPoint: /v1/mimoto/binding-otp
```
Comment (mohanachandran-s) [inline apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml:677]: Endpoint shouldn't be changed here

**Review by mohanachandran-s — CHANGES_REQUESTED** [review]

File: apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml (hunk 2 above)
Comment (santhoshsunder) [reply apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml]: changes done

File: apitest/src/main/resources/mimoto/AuthUnLock/createAuthUnLock.yml (hunk 4 above)
Comment (santhoshsunder) [reply apitest/src/main/resources/mimoto/AuthUnLock/createAuthUnLock.yml]: changes done

File: apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml (hunk 3 above)
Comment (santhoshsunder) [reply apitest/src/main/resources/mimoto/AuthLock/createAuthLock.yml]: changes done

File: apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml (hunk 5 above)
Comment (santhoshsunder) [reply apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml]: changes done

File: apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml (hunk 6 above)
Comment (santhoshsunder) [reply apitest/src/main/resources/mimoto/BindingOtp/BindingOtp.yml:677]: changes done

**Review by mohanachandran-s — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**